File: Practica7/Juego1/Java/AdivinaNumero.java
import java.util.Scanner;
import java.util.Random;
class AdivinaNumero extends Juego {
    int nroAdivina;
    Scanner sc = new Scanner(System.in);
    Random r = new Random();
    public void juega() {
        reiniciaPartida();
        nroAdivina = r.nextInt(11);
        System.out.println("el numero esta entre 0 y 10");
        while (true) {
            System.out.print("n: ");
            int n = sc.nextInt();
            if (n == nroAdivina) {
                System.out.println("Boeeee, hacker w, ganaste");
                actualizaRecord();
                break;
            } else {
                if (quitaVida()) {
                    if (n < nroAdivina) {
                        System.out.println("mas grande");
                    } else {
                        System.out.println("mas peque");
                    }
                } else {
                    System.out.println("XDDDDD, PERDISTE, WUJAJAJAJ 😂");
                    break;
                }
            }
        }
    }
}

File: Practica7/Juego1/Python/AdivinaNumero.py
from Juego import Juego
import random
class JuegoAdivinaNumero(Juego):
    def __init__(self):
        super().__init__()
    def juega(self):
        self.reiniciaPartida()
        self.a = random.randint(0, 10)
        print("Escoge un numero entre el 0 y 10, pero ojo, tienes 3 cidas w")
        while True:
            try:
                n = int(input("Mete un número: "))
                if n == self.a:
                    print("boeeeee, le atinaste :) ")
                    self.actualizaRecord()
                    break
                else:
                    if self.quitaVida():
                        if n < self.a:
                            print("tu numero es mas grande")
                        else:
                            print("tu numero es mas peque")
                    else:
                        print("XDD, Perdiste w el numero q te asignaron era", self.a)
                        break
            except:
                print("Viejo, no entiendes, dije del 0 al 10 q es n", n)